src/webrtc.ts

```diff
@@ -234,11 +234,19 @@ export class WebrtcExtension extends EventEmitter {
     //   this.emit('send', data);
     // });
 
-    this.client._stanzaio.on('jingle:outgoing', (session) => {
+    this.client._stanzaio.on('jingle:outgoing', (session: any) => {
```

Comment:
NIT: this may be able to be typed as `JingleSession` from `import JingleSession from 'stanza/jingle/Session'`